lib/services/nativescript-migration-service.ts

```diff
@@ -74,6 +74,10 @@ export class NativeScriptMigrationService implements IFrameworkMigrationService
 				return framework.displayName;
 			}
 
+			if(_.any(this.getObsoleteVersions().wait(), obsoleteVersion => obsoleteVersion === version)) {
+				this.$errors.failWithoutHelp(`Your project is using obsolete framework version - ${version}. You can still build your project, but you cannot migrate to a newer version.`);
```

Comment:
Your project targets NativeScript ${version}. This version is obsolete and cannot be migrated from the command line. If you want to migrate your project, create a new project and copy over your code and resources.